.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,51 @@
+name: UnitTests
+
+on:
+  pull_request:
+    branches: 
+     - "**"
+
+jobs:   
+  run-unit-tests:
+
+    runs-on: ubuntu-latest
+    container: compucorp/civicrm-buildkit:1.0
+    env:
+      CIVICRM_CORE: 5.24.6
+      SHOREDITCH_TAG: 0.1.0-alpha.37
```

Comment:
I'm not sure if this should really be hardcoded. I feel it should be master. Better ask @deb1990 or @reneolivo

Reply:
@reneolivo  @deb1990  could you please advise if we should always install Shoreditch from the master branch before installing CiviCase for testing feature branch?

Reply:
@davialexandre  Civicase is being developed with Shoredtich version  01.0-alpha.37 as it is on Case-Master branch on compuclient. In the future, CiviCase team should be responsible to maintain the version of shoredtich on the workflow.

Reply:
I think using Shoreditch's `master` branch is fine for running Javascript Unit Tests since we don't test any styles. That's what we use Backstop for and I don't think that's covered by these actions.

If we target `master` we won't forget to update this value when the shoreditch version changes.

Reply:
@davialexandre  in this case, I will update to use shoredtich master master branch as we only need to install it so we could install civicase and as @reneolivo  mentioned even we need to test Javascript. it will be no issue too.